inspirehep/utils/bibtex.py

```diff
@@ -24,7 +24,7 @@
 
 import re
 
-from inspirehep.dojson.utils import force_force_list
+from inspirehep.utils.helpers import force_force_list
```

Comment:
Order: should go after `inspirehep.utils` and before `inspirehep.utils_record` (which should go before `record_getter`...)